packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -99,6 +225,17 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
 
   void _flushPointerEventQueue() {
     assert(!locked);
+
+    if (resamplingEnabled) {
+      _resampler.addOrDispatchAll(_pendingPointerEvents);
+      _resampler.sample(samplingOffset);
```

Comment:
Does this mean that, when the pending event queue in the resampler is not empty, but there are no more future inputs, these pending events will stay in the queue and will not be processed, since _flushPointerEventQueue is not called?
I realize this when I'm trying to move resampler to handlePointerEvent.
cc @liyuqian

Reply:
No, the `_Resampler` will schedule `_handleSampleTimeChanged` after frame callback which will call `_flushPointerEventQueue`. See: https://github.com/flutter/flutter/pull/60558/files/b7802f69f91e1b5466d6b5b58e98193160964ca5#diff-6305361e0c7677f0eebfb05dfcbe0336